docs/plugins/notice.md

```diff
@@ -115,7 +115,7 @@ def keep_alive(self):
     return {}
 
 # With full control, multiple templates, still connected to chat.
-@route("/complex_page/<page_id:int>", method=POST)
+@route("/complex_page/<page_id:int>", method='POST')
```

Comment:
Double quotes are used everywhere else from what I can tell... Should this follow that convention?